mattermost.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python2
+#!/usr/bin/python3
```

Comment:
Use the following to support both python2 and python3.

    #!/usr/bin/env python